bindings/cpp/CMakeLists.txt

```diff
@@ -0,0 +1,66 @@
+cmake_minimum_required(VERSION 3.12)
+cmake_policy(VERSION 3.12)
+
+project(PortAudio LANGUAGES CXX)
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
These C++ files haven't even been substantially modified since the first Git commit in 2008.

Reply:
I've added this with the possibility in mind to modernize the code. 
The code should be at least modernized to c++11 and since there are no relevant breaking changes between 11 and 14 but enourmous improvements, it is better to skip c++11.

Another sidefact, llvm is based on cpp14 and msvc does not even support c++11 anymore. The minimal supported version is c++14.

Reply:
Modernizing the code sounds like a good idea, but until that's done, there's no reason to reduce compatibility.